packages/client/src/pages/LeaderBoard/LeaderBoard.tsx

```diff
@@ -1,3 +1,61 @@
-export function LeaderBoard() {
-  return <h1>LeaderBoard</h1>
-}
+import Title from '@/components/Title'
+import styles from './leaderBoard.module.css'
+import { FC } from 'react'
+import { getCorrectForm } from '@/utils/helpers/convertWord'
```

Comment:
Я немного косякнул при создании алиасов, зачем-то добавил `@`, но он думаю лишний, jest на него ругается тоже. Уберу в следующем PR, чтобы не путались. И поправь во всех файлах, пожалуйста, чтобы не было обращений таких
```suggestion
import { getCorrectForm } from '@utils/helpers/convertWord'
```